apps/mull-ui/src/app/pages/create-event/location-autocomplete/location-autocomplete-modal.spec.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { fireEvent, render, waitFor } from '@testing-library/react';
+import '@testing-library/jest-dom';
+
+import LocationAutocompleteModal from './location-autocomplete-modal';
+
+const mockFormik = {
+  values: { location: 'test string' },
+  setFieldValue: { location: 'value' },
+  touched: { location: false },
+  errors: { location: false },
+};
+
+describe('location autocomplete modal', () => {
+  it('should render successfully', () => {
+    const { baseElement } = render(<LocationAutocompleteModal formik={mockFormik} />);
+    expect(baseElement).toBeTruthy();
+  });
+
+  it('should match snapshot', () => {
+    const tree = renderer.create(<LocationAutocompleteModal formik={mockFormik} />).toJSON();
+    expect(tree).toMatchSnapshot();
+  });
+
+  it('should open autocomplete modal', async () => {
+    const utils = render(<LocationAutocompleteModal formik={mockFormik} />);
+    const autocompleteModal = utils.container.querySelector('#location');
+    await waitFor(() => {
+      fireEvent.click(autocompleteModal);
+    });
+  });
```

Comment:
This isn't testing anything, there should be an `expect` to check that the modal was opened

Reply:
Addressed